### ISSUE TITLE
Missing images folder

### ISSUE THREAD

**Issue by pedroestevaodev** [pepiuox/PHP-GrapesJS#11]
The system cannot find the img folder that should be in Path/dist/

**Comment by pepiuox** [comment]
> 
> 
> The system cannot find the img folder that should be in Path/dist/

now it's corrected, is ../uplodas

**Comment by EGSolda** [comment]
I'm having the same problem. The system cannot find the images that should be in the directory `<?php echo $base ?>dist/img...`
When I went to check, I realized that the directory doesn't really exist. Did you forget to upload?

![Captura de tela 2021-09-15 101921](https://user-images.githubusercontent.com/90622128/133441968-21f189ad-4eb4-437a-96fa-472f998c41ed.png)

![Captura de tela 2021-09-15 102905](https://user-images.githubusercontent.com/90622128/133442255-37bec39e-021f-4226-9411-80c35c28fa64.png)

![Captura de tela 2021-09-15 102111](https://user-images.githubusercontent.com/90622128/133441972-1850d33d-4bc6-4999-aa32-5f747175aab1.png)

![Captura de tela 2021-09-15 102233](https://user-images.githubusercontent.com/90622128/133441974-33c7df22-a493-40d5-9f6a-0eff08053836.png)
